speechbrain/decoders/seq2seq.py

```diff
@@ -644,15 +650,6 @@ def forward(self, enc_states, wav_len):  # noqa: C901
             if t < min_decode_steps:
                 log_probs[:, self.eos_index] = self.minus_inf
 
-            # Set the eos prob to minus_inf when it doesn't exceed threshold.
-            if self.using_eos_threshold:
```

Comment:
I modify this part. We should check the eos threshold after all scorers (attn scorer, ctc scorer and lm scorer), or the emission of eos will only be based on the output of attn scorer.